projects/MCBI.yaml

```diff
@@ -0,0 +1,12 @@
+Name: MCBI
+Description: processing images with Freesurfer
+Department: Martinos Center for Biomedial Imaging
+FieldOfScience: Health
```

Comment:
Sorry, merged this too soon. Could you change this to one of
- `Biomedical research`
- `Medical Imaging`
- `Medical Sciences`
?
whichever is most appropriate?

The list of fields of science we should use is in the left column of [this spreadsheet](https://topology.opensciencegrid.org/nsfscience/csv).

Reply:
Done in #1062